src/sentry/features/__init__.py

```diff
@@ -67,7 +67,7 @@
 default_manager.add("organizations:dashboards-manage", OrganizationFeature, True)
 default_manager.add("organizations:data-forwarding", OrganizationFeature)
 default_manager.add("organizations:discover", OrganizationFeature)
-default_manager.add("organizations:discover-arithmetic", OrganizationFeature, True)
+default_manager.add("organizations:discover-top-events", OrganizationFeature, True)
```

Comment:
Should we just add it to `conf/server.py` as well since the feature flag is being introduced in this PR?

Reply:
:+1: added in 2759ff7